lib/src/widgets/shared_resources_table.dart

```diff
@@ -0,0 +1,170 @@
+                  padding: const EdgeInsets.fromLTRB(0, 5, 0, 0),
+                  width: cWidth,
+                  child: SelectableText(
+                    index as String,
+                    style: const TextStyle(fontWeight: FontWeight.bold),
+                  ),
+                )),
+                DataCell(
+                  Row(
+                    children: [
+                      Expanded(
+                        child: Text(
+                          getDateTime(sharedResMap[index]
+                              [PermissionLogLiteral.logtime] as String),
+                        ),
+                      ),
+                    ],
+                  ),
+                ),
+                buildDataCell(
+                    sharedResMap[index][PermissionLogLiteral.owner] as String),
+                buildDataCell(sharedResMap[index][PermissionLogLiteral.granter]
+                    as String),
+                buildDataCell(sharedResMap[index]
+                    [PermissionLogLiteral.permissions] as String),
+                DataCell(
+                  IconButton(
+                      icon: const Icon(
+                        Icons.visibility,
+                        size: 24.0,
+                        color: Colors.blueAccent,
+                      ),
+                      onPressed: () async {
+                        // Get file content
+                        final fileContent =
+                            await readExternalPod(index, context, parentWidget);
+
+                        if (fileContent != null) {
+                          if (!context.mounted) return;
+                          await showDialog(
+                              context: context,
+                              builder: (ctx) => AlertDialog(
+                                    title: const Text('File content'),
+                                    content: Stack(
+                                      alignment: Alignment.center,
+                                      children: <Widget>[
+                                        Container(
+                                          width: double.infinity,
+                                          height: 300,
+                                          decoration: BoxDecoration(
+                                            borderRadius:
+                                                BorderRadius.circular(15),
+                                          ),
+                                          child: Text(fileContent as String),
+                                        ),
+                                      ],
+                                    ),
+                                    actions: [
+                                      TextButton(
+                                          onPressed: () {
+                                            // Close the dialog
+                                            Navigator.of(context).pop();
```

Comment:
I got a null pointer exception here which seems to be related to the use of `context` (when clicking the `Ok` button in the `File Content` dialog.

Replacing `context` with `ctx` seems to fix the issue.